frontend/src/components/cronjob/Details.tsx

```diff
@@ -12,7 +18,30 @@ export default function CronJobDetails() {
   const { t, i18n } = useTranslation('glossary');
 
   const [jobs, jobsError] = Job.useList();
+  const [cronJob, cronJobError] = CronJob.useGet(name, namespace);
+  const [isCronSuspended, setIsCronSuspended] = useState(false);
+  const [isCheckingCronSuspendStatus, setIsCheckingCronSuspendStatus] = useState(true);
+  const dispatch = useDispatch();
 
+  function applyFunc(newItem: KubeObjectInterface) {
+    return apply(newItem)
+      .then(value => {
+        console.log('success value is ', value);
+        setIsCronSuspended(!isCronSuspended);
```

Comment:
A best practice for doing this, due to how the values may differ from expected, is to do:
`setIsCronSuspended(isCronSuspended => !isCronSuspended);`

Reply:
Actually is this even necessary? AFAIK, the proper value here will be done when the `cronJob` object gets updated.

Reply:
yupp you are right we don't need this here